src/Layers/xrRender/ParticleGroup.h

```diff
@@ -72,7 +72,7 @@ class ECORE_API CPGDef
 
 class ECORE_API CParticleGroup : public dxParticleCustom
 {
-    const CPGDef* m_Def;
+    CPGDef* m_Def;
```

Comment:
Why remove const?
And we can just initialize the variable in the class declaration:
```suggestion
    const CPGDef* m_Def{};
```

Reply:
It's even better and more convenient this way